modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/schema/fieldValidation/getAllContentsFieldsValidation.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {array, object} from 'yup';
+
+import {validateDocument} from '../constants/validateDocument';
+
+export const getAllContentsFieldsValidation = {
```

Comment:
```suggestion
export const allContentsFieldsValidation = {
```

We use get just for functions